src/core/iomgr/tcp_windows.c

```diff
@@ -266,7 +266,6 @@ static grpc_endpoint_write_status win_write(grpc_endpoint *ep,
   WSABUF *buffers = local_buffers;
 
   GPR_ASSERT(nslices != 0);
-  GPR_ASSERT(GPR_SLICE_LENGTH(slices[0]) != 0);
```

Comment:
I feel this is a leftover from former, depreciated Posix port. I can't find the equivalent in the posix code anymore, meaning this is indeed no longer relevant anymore. I feel we also want to remove the assert above on nslices != 0 since it doesn't exist in posix either.